File: Pertemuan 5/BelajarOperator.java
import java.util.Scanner;

public class BelajarOperator {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Masukkan nilai A = ");
        double a = scanner.nextDouble();
        System.out.print("Masukkan nilai B = ");
        double b = scanner.nextDouble();
        System.out.println("A == B = " + (a==b));
        System.out.println("A != B = " + (a!=b));
        System.out.println("A > B = " + (a>b));
        System.out.println("A < B = " + (a<b));
        System.out.println("A >= B = " + (a>=b));
        System.out.println("A <= B = " + (a<=b));
        System.out.println();
        boolean hasil = (a < 5 && b > 2);
        System.out.println("Hasil dari A < 5 && B > 2 = " + hasil);
        System.out.println();
        System.out.println("Hasil A < 5 || B > 2 = " + hasil);
        System.out.println("hasil !(A < 5 && B > 2) = " + hasil);
    }
}

File: Pertemuan 5/belajaroperator.py
a = input("Masukkan nilai A = ")
b = input("Masukkan nilai B = ")
x = True
y = False
print("A == B = " + str(a==b))
print("A != B = " + str(a!=b))
print("A > B = " + str(a>b))
print("A < B = " + str(a<b))
print("A <= B = " + str(a<=b))
print("A >= B = " + str(a>=b))
print(" ")
x = int(a) < 5 and (b) > 4
print("A < 5 and B > 4 = " + str(x))
print("A < 5 or B > 4 = " + str(int(a) <5 or int(b) >4))
print("A < 5 not B > 2 = " + str(not(int(a) <5)))